components/usage/pkg/server/server.go

```diff
@@ -158,7 +160,7 @@ func registerGRPCServices(srv *baseserver.Server, conn *gorm.DB, stripeClient *s
 	if stripeClient == nil {
 		v1.RegisterBillingServiceServer(srv.GRPC(), &apiv1.BillingServiceNoop{})
 	} else {
-		v1.RegisterBillingServiceServer(srv.GRPC(), apiv1.NewBillingService(stripeClient, conn, ccManager))
+		v1.RegisterBillingServiceServer(srv.GRPC(), apiv1.NewBillingService(stripeClient, conn, ccManager, &cfg.StripePrices))
```

Comment:
I'd recommend passing in the `StripePrices` without a pointer. This "ensures" that the handler can't modify them.

Reply:
Aha, got it.